README.md

```diff
@@ -30,9 +30,9 @@ To stop everything, just run `docker-compose down`.
 ## Troubleshooting & Maintenance
 
 ### Resetting Catalog Data
-To reset the catalog and data, remove the `postgres` and `warehouse` directories.
+To reset the data, remove the `warehouse` directories.
```

Comment:
We no longer need this `warehouse` directory if we're using MinIO right? Should we get rid of it completely and just let the MinIO volume handle persisting the data between sessions?

Reply:
You're right, thanks! Updated the docs